docs/guide/personalization.md

```diff
@@ -94,15 +94,15 @@ In order to tell the recommender to use this user and password to request resour
 you can configure this as follows (an example file is available in the bundle under `Resources/config/default_settings.yml`):
 
 ``` yaml
-# Export folder authentication method:
-# basic - password generated by script
-# user - login and passwords set by user
-# none - no authentication !Warning - allow users to download content without permission
-ez_recommendation.export.users_authentication.method: user
-# Credentials for user authentication
-# assure to use double quotes, otherwise an int is compared with a string during authentication phase
-ez_recommendation.export.users_authentication.login: '<customerID>'
-ez_recommendation.export.users_authentication.password: '<licenseKey>'
+ezrecommendation:
+    system:
+        <siteaccess>:
+            export:
+                authentication:
+                    method:
+                    login: 1111
+                    password: 12345-12345-1234-1234-1234
```

Comment:
Default authentication method should be "basic"

I think we should also add info about using "user" method. If we choose this method then login must be the same as customer_id and password must be the same as license_key. I think it will be more readable if we set login to 12345 and password = 1234-5678-9012-3456-7890 (as we have for customer_id and license_Key)